google/cloud/storage/blob.py

```diff
@@ -3018,6 +3019,18 @@ def custom_time(self):
         if value is not None:
             return _rfc3339_to_datetime(value)
 
+    @custom_time.setter
+    def custom_time(self, value):
+        """Set the custom time for the object.
+
+        See https://cloud.google.com/storage/docs/json_api/v1/objects
+
+        :type value: :class:`datetime.datetime`
+        :param value: (Optional) Set the custom time of blob. Datetime object parsed
+                                 from RFC3339 valid timestamp.
+        """
+        self._properties["customTime"] = _datetime_to_rfc3339(value)
```

Comment:
If `value` is None, the conversion will fail.  Suggest:
```suggestion
        if value is not None:
            value = _datetime_to_rfc3339(value)
            
        self._properties["customTime"] = value
```